elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreTestHarness.java

```diff
@@ -6,42 +6,33 @@
 package com.yahoo.elide.datastores.aggregation;
 
 import com.yahoo.elide.core.DataStore;
-import com.yahoo.elide.core.EntityDictionary;
 import com.yahoo.elide.core.datastore.test.DataStoreTestHarness;
-import com.yahoo.elide.datastores.aggregation.example.Country;
-import com.yahoo.elide.datastores.aggregation.example.Player;
 import com.yahoo.elide.datastores.aggregation.example.PlayerStats;
-import com.yahoo.elide.datastores.aggregation.example.PlayerStatsView;
-import com.yahoo.elide.datastores.aggregation.example.SubCountry;
-import com.yahoo.elide.datastores.aggregation.example.VideoGame;
 import com.yahoo.elide.datastores.aggregation.metadata.MetaDataStore;
+import com.yahoo.elide.datastores.aggregation.queryengines.sql.SQLQueryEngineFactory;
+import com.yahoo.elide.datastores.jpa.JpaDataStore;
+import com.yahoo.elide.datastores.jpa.transaction.NonJtaTransaction;
 import com.yahoo.elide.datastores.multiplex.MultiplexManager;
 
 public class AggregationDataStoreTestHarness implements DataStoreTestHarness {
-    private QueryEngineFactory queryEngineFactory;
+    private SQLQueryEngineFactory queryEngineFactory;
 
-    public AggregationDataStoreTestHarness(QueryEngineFactory queryEngineFactory) {
+    public AggregationDataStoreTestHarness(SQLQueryEngineFactory queryEngineFactory) {
         this.queryEngineFactory = queryEngineFactory;
     }
 
     @Override
     public DataStore getDataStore() {
-        MetaDataStore metaDataStore = new MetaDataStore();
-        AggregationDataStore aggregationDataStore = new AggregationDataStore(queryEngineFactory, metaDataStore) {
-            @Override
-            public void populateEntityDictionary(EntityDictionary dictionary) {
-                dictionary.bindEntity(PlayerStats.class);
-                dictionary.bindEntity(Country.class);
-                dictionary.bindEntity(SubCountry.class);
-                dictionary.bindEntity(PlayerStatsView.class);
-                dictionary.bindEntity(Player.class);
-                dictionary.bindEntity(VideoGame.class);
-                super.populateEntityDictionary(dictionary);
-            }
-        };
+        MetaDataStore metaDataStore = new MetaDataStore(PlayerStats.class.getPackage());
+        AggregationDataStore aggregationDataStore = new AggregationDataStore(queryEngineFactory, metaDataStore);
+
+        DataStore jpaStore = new JpaDataStore(
+                () -> { return queryEngineFactory.getEmf().createEntityManager(); },
+                (entityManager) -> { return new NonJtaTransaction(entityManager); }
+        );
 
         // meta data store needs to be put at first to populate meta data models
-        return new MultiplexManager(metaDataStore, aggregationDataStore);
+        return new MultiplexManager(jpaStore, metaDataStore, aggregationDataStore);
```

Comment:
So the JPA models are not managed by metaDataStore?

Reply:
Correct.  JPA models are managed by JPA Store.  MetaDataStore only manages a small subset of entities (Table, Column, etc).

Reply:
I mean the data models in JPA store. Would those models be added into the metadatastore? For example, there is a JPA model call `People`, would be schema of `People` be stored in metadatastore?

Reply:
MetadataStore has two dictionaries - the one it exposes to Elide core (which just has Table, Column, etc), and other dictionary which contains the entities it adds to the metadata.  That is populated by scanning the class path.